numpy/_core/strings.py

```diff
@@ -573,15 +578,13 @@ def center(a, width, fillchar=' '):
     Return a copy of `a` with its elements centered in a string of
     length `width`.
 
-    Calls :meth:`str.center` element-wise.
-
     Parameters
     ----------
     a : array_like, with `np.bytes_` or `np.str_` dtype
 
-    width : int
+    width : array_like, with any integer dtype
         The length of the resulting strings
```

Comment:
I think this should this be "The *minimum* length of ..."? (same for others below)

Reply:
Would something like this be better?

> The length of the resulting string, unless `width < str_len(a)`

Reply:
That's fine too!